cli-test/spec/automated_test_cases/performance_test.rb

```
@@ -135,21 +136,21 @@
     puts "oxen checkout main command took: #{checkout_main_branch_time} seconds"
     expect(checkout_main_branch_time).to be < 100.0
 
-    run_command_and_stop('mkdir files')
+    FileUtils.mkdir_p('files')
 
-      directory_path = 'tmp/aruba/test-small-repo'
+      directory_path = File.join('tmp', 'aruba', 'test-small-repo') 
```

RuboCop (1.69.1)

[warning] Useless assignment to variable - `directory_path`.

(Lint/UselessAssignment)